plugins/Referrers/DataTable/Filter/UrlsForSocial.php

```diff
@@ -14,24 +14,6 @@
 
 class UrlsForSocial extends BaseFilter
 {
-    /**
-     * @var bool
-     */
-    private $sortRecursive;
-
-    /**
-     * Constructor.
-     *
-     * @param DataTable $table The table to eventually filter.
-     * @param bool $sortRecursive Whether to sort recursive or not
-     */
-    public function __construct($table, $sortRecursive)
```

Comment:
That parameter is passed to the filter in two places:

https://github.com/matomo-org/matomo/blob/2990ce03f586704c4c131d29c4605dcfebe1472a/plugins/Referrers/API.php#L682

https://github.com/matomo-org/matomo/blob/2990ce03f586704c4c131d29c4605dcfebe1472a/plugins/Referrers/API.php#L924

I would say we update those as well to not pass the parameter.